template.go

```diff
@@ -149,6 +156,11 @@ func AddTemplateExt(ext string) {
 	beeTemplateExt = append(beeTemplateExt, ext)
 }
 
+func AddViewPath(viewPath string) error {
+	beeViewPathTemplates[viewPath] = make(map[string]*template.Template)
```

Comment:
It's a tradeoff. If we add mutex here all the template render will add lock. I think maybe we just add the documents for the caller.